packages/asc-ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -40,10 +41,12 @@ const Checkbox = React.forwardRef<
     return (
       <CheckboxWrapperStyle
         {...{ className, disabled, focus, checked, error }}
+        // eslint-disable-next-line no-nested-ternary
+        aria-checked={checked ? (indeterminate ? 'mixed' : true) : false}
```

Comment:
Do we need to add these ARIA attributes? `CheckboxStyle` is already an `<input type="checkbox">`, it should provide all the information that is needed here already. It also has support for setting the `indeterminate` state, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Indeterminate_state_checkboxes